waffle_configs/bedrock-prod.env

```diff
@@ -23,7 +23,7 @@ SWITCH_FIREFOX_NEW_OPTIMIZELY=off
 SWITCH_FIREFOX_NEW_SCENE2_OPTIMIZELY=off
 SWITCH_FIREFOX_UPDATE_NOTIFICATION=on
 SWITCH_FIREFOX_UPDATE_NOTIFICATION_MODAL=on
-SWITCH_FIRSTRUN_COPY_EXPERIMENT=on
+SWITCH_FIRSTRUN_COPY_EXPERIMENT=off
```

Comment:
Can you make this change to `bedrock-stage.env` as well so they are in sync? Thanks!